source/reference/minio-mc-admin/mc-admin-prometheus-generate.rst

```diff
@@ -110,9 +206,65 @@ The output resembles the following:
 .. code-block:: shell
 
    scrape_configs:
-   - job_name: minio-job-bucket
+   - job_name: minio-job-replication
      bearer_token: [auth token]
-     metrics_path: /minio/v2/metrics/bucket
+     metrics_path: /minio/metrics/v3/bucket/replication/mybucket
+     scheme: https
+     static_configs:
+     - targets: [`localhost:9000`]
+
+Generate a v3 config for bucket API metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example generates a scrape configuration for v3 API metrics for bucket ``mybucket``:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin prometheus generate ALIAS api --bucket mybucket --api-version v3
+
+- Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+The output resembles the following:
+
+.. code-block:: shell
+
+   scrape_configs:
+   - job_name: minio-job-api
+     bearer_token: [auth token]
+     metrics_path: /minio/metrics/v3/bucket/api/mybucket
+     scheme: https
+     static_configs:
+     - targets: [`localhost:9000`]
+
+Generate a default metrics v2 config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, :mc-cmd:`mc admin prometheus generate` generates a scrape configuration for v2 cluster metrics:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin prometheus generate ALIAS
+
+- Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+The output resembles the following:
+
+.. code-block:: shell
+
+   scrape_configs:
+   - job_name: minio-job
+     bearer_token: [auth token]
+     metrics_path: /minio/v2/metrics
      scheme: http
      static_configs:
      - targets: ['localhost:9000']
+
+To generate a configuration for another metric type, specify the type.
```

Comment:
I'd drop this line, since the example is for default metrics.
Or move this to part of the a separate example (as suggested for the next line).

Reply:
added another example